ddtrace/contrib/anthropic/async_message.py

```diff
@@ -0,0 +1,86 @@
+import json
+import sys
+
+from ddtrace.contrib.trace_utils import with_traced_module
+from ddtrace.internal.logger import get_logger
+from ddtrace.internal.utils import get_argument_value
+
+from .utils import _extract_api_key
+from .utils import handle_non_streamed_response
+
+
+log = get_logger(__name__)
+
+
+@with_traced_module
```

Comment:
I think we can just keep this method in `patch.py` to keep the traced methods in the same file (if we had to trace multiple methods I wouldn't mind but there's only two methods)